components/org.wso2.carbon.identity.oauth.endpoint/src/main/java/org/wso2/carbon/identity/oauth/endpoint/authz/OAuth2AuthzEndpoint.java

```diff
@@ -4183,4 +4197,13 @@ private void addUserAttributesToCache(SessionDataCacheEntry sessionDataCacheEntr
                 new DeviceAuthorizationGrantCacheEntry(sessionDataCacheEntry.getLoggedInUser().getUserAttributes());
         DeviceAuthorizationGrantCache.getInstance().addToCache(cacheKey, cacheEntry);
     }
+
+
+    private boolean isFapiConformant(String clientId) throws InvalidRequestException {
+        try {
```

Comment:
Add new line

Reply:
Fixed in d04102128137314d9c789e0d4a4bf9921b1bff01